### PR TITLE
HH-145 refactor: move event location and graphql files to the component package

### PR DESCRIPTION
**Pull request by nenonja**
This is a refactoring before implementing cookie disable/enable feature for palvelukartta cookies. I didn't want to do a duplicate code changes to identical EventLocation component which resided in both hobbies and events apps.
So I decided to move EventLocation.tsx and dependent files to component package. Because it's dependent on graphql generated types, I moved the generated types there as well along with some other files like eventUtils and some others.

Basically this 95% of changed imports. Also `Keyword.tsx` is renamed to `KeywordButton.tsx` because graphql has `Keyword`-type which caused confict.

- also: Fixed jest timeouts to be 50 seconds on CI build

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

File: apps/events-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const EventKeywords: React.FC<Props> = ({
         />
       )}
       {showIsFree && freeEvent && (
-        <Keyword
+        <KeywordButton
```
Comment (melniiv) [inline apps/events-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx:101]: We should think about the naming, because the spec is that it might be not even a "button". I would say its clickable or non clickable tag, and also, we have to consider to replace it with hcrc tag component.

**Review by melniiv — APPROVED** [review]
LGTM, just small comments. Also, would be nice to improve finally importing to use some aliases, otherwise those kind of imports like `'../../../clients/eventsApolloClient` looks too long

**Comment by nenonja** [comment]
> LGTM, just small comments. Also, would be nice to improve finally importing to use some aliases, otherwise those kind of imports like `'../../../clients/eventsApolloClient` looks too long

Yes I have been thinking the same, that the imports should be refactored to use root paths instead of relative paths. Relative paths makes the refactoring and search & replacing harder.

File: apps/events-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx (hunk 1 above)
Comment (nenonja) [reply apps/events-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx:101]: Perhaps I could rename it to `KeywordTag`
